.github/workflows/release.hutch-agent.yml

```diff
@@ -10,9 +10,7 @@ on:
     types:
       - closed
     branches:
-      - release/**
-    paths:
-      - /app/HutchAgent/**
+      - main
```

Comment:
we still need the workflow (or its jobs) to be conditional on the compare branch ref matching `release/agent-**`